pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -207,6 +209,22 @@ def test_OA_integer_to_binary(self):
                 self.assertAlmostEqual(
                     value(model.objective.expr), model.optimal_value, places=1)
 
+    def test_OA_partition_obj_nonlinear_terms(self):
+        """Test the outer approximation decomposition algorithm (partition_obj_nonlinear_terms)."""
+        with SolverFactory('mindtpy') as opt:
+            print('\n Solving 8PP problem with Outer Approximation')
```

Comment:
Should this message be sent out through a logger?

Reply:
I just removed it.